<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>proizvodDodaj</class>
 <widget class="QDialog" name="proizvodDodaj">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>481</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>855</width>
    <height>481</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>855</width>
    <height>481</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Aplikacija Pekara | Dodavanje Proizvoda</string>
  </property>
  <widget class="QPushButton" name="izlazDugme">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>400</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Izlaz</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_naziv_dodaj">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="dodajDugme">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>400</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Dodaj u magacin</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nabavnaCena">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_kolicina_dodaj">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="magacinIzbor">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>300</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Gram</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Kilogram</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Komad</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tona</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="SekvencaLabela2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Dodavanje proizvoda:</string>
   </property>
  </widget>
  <widget class="QLabel" name="KolicinaLabela2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Koliƒçina:</string>
   </property>
  </widget>
  <widget class="QLabel" name="jedMereLabela2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Jedinica mere:</string>
   </property>
  </widget>
  <widget class="QLabel" name="nCenaLabela">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nabavna cena:</string>
   </property>
  </widget>
  <widget class="QLabel" name="NazivLabela2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Naziv:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
